refactor(examples): Update `flower-authentication` example to use `flwr supernode create/list` (#5998)
adap/flower: examples/flower-authentication/README.md

python prepare_dataset.py
```

### Pre-registering SuperNodes

Before connecting the `SuperNodes` we need to register them with the `SuperLink`. This means we'll tell the `SuperLink` about the identities of the `SuperNodes` that will be connected. We do this by sending to it the public keys of the `SuperNodes` that we want the `SuperLink` to authorize.

Let's pre-register the first `SuperNode`. The command below will send the public key to the `SuperLink` defined in the `my-federation` federation in the `pyproject.toml`.

```shell
flwr supernode create keys/client_credentials_1.pub . my-federation
# It will print something like:
# Loading project configuration...
# Success
# ✅ Node 16019329408659850374 created successfully.
```

> Then, we pre-register the second `SuperNode` using the other public key:

Reviewer: ```suggestion
Then, we register the second `SuperNode` using the other public key:
```

```shell

> flwr supernode create keys/client_credentials_2.pub . my-federation

Reviewer: ```suggestion
flwr supernode register keys/client_credentials_2.pub . my-federation
```

# It will print something like:
# Loading project configuration...
# Success

> # ✅ Node 8392976743692794070 created successfully.

Author: ```suggestion
# ✅ SuperNode 8392976743692794070 registered successfully.
```

    --clientappio-api-address="0.0.0.0:9095"
```

Now that you have connected the `SuperNodes`, you should see them with status `online`:

```shell
flwr supernode list . my-federation
📄 Listing all nodes...
┏━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┓
┃       Node ID        ┃   Owner    ┃ Status  ┃ Elapsed  ┃   Status Changed @   ┃
┡━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━┩
│ 16019329408659850374 │ sys_noauth │ online  │ 00:00:30 │ 2025-10-13 13:40:47Z │
├──────────────────────┼────────────┼─────────┼──────────┼──────────────────────┤
│ 8392976743692794070  │ sys_noauth │ online  │ 00:00:22 │ 2025-10-13 13:52:21Z │
└──────────────────────┴────────────┴─────────┴──────────┴──────────────────────┘
```

If you generated more than 2 client credentials, you can add more clients by opening new terminal windows and running the command

> above. Don't forget to specify the correct client private and public keys for each client instance you created.

Reviewer: ```suggestion
above. Don't forget to specify the correct client private key for each client (SuperNode) you created.
```
